modules/ivc_champva/app/services/ivc_champva/file_uploader.rb

```diff
@@ -48,6 +50,8 @@ def insert_form(file_path, response_status)
         s3_status: response_status,
         pega_status:
       )
+
+      monitor.track_insert_form(@metadata['uuid'], @form_id)
```

Comment:
Log when a form is submitted/created and include the form UUID so that our DD plot can connect form submissions to Pega updates.